jobs/ci-run/integration/gen/test-smoke.yml

```diff
@@ -191,7 +191,7 @@
 
 - job:
     name: test-smoke-test-deploy-aws
-    node: ephemeral-focal-small-amd64
+    node: ephemeral-focal-8c-32g-amd64
```

Comment:
this needs to be small - we don't need a large instance to run the juju cli

Reply:
Had to be bumped up for the moment because they were being OOM killed.